hub/powertoys/run.md

```diff
@@ -117,7 +111,7 @@ The Windows System Commands plugin provides a set of system level actions that c
 | `Sleep` | Puts the computer to sleep | |
 | `Hibernate` | Hibernates the computer | |
 | `Empty Recycle Bin` | Empties the recycle bin | |
-| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
+| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br>(Requires administrative permissions.) |
```

Comment:
Dont remove / as this is not the correct html code. @crutkas want me to write it with / in an older PR.